Renamed absTime in LinkBB to absTP

Manarnz opened the pull request:
This fixes the issue #493

Manarnz committed:
Renamed absTime in LinkBB to absTP

this fixes the issue #493

YesugenB commented on leaf-ui/rappid-extensions/AssignmentsTable.js:
```diff
@@ -383,7 +383,7 @@ var LinkRelationshipView = Backbone.View.extend({
         '<td class= "link-source"></td>',
         '<td class= "link-dest"></td>',
-        '<td><input class="linkAbsRelation" type="number" name="sth" value= <%- absTime %> > </td>',
+        '<td><input class="linkAbsRelation" type="number" name="sth" value= <%- absTP %> > </td>',
```
Wondering if we can remove the `name="sth"` as I don't think we really need it? Not sure though, what does it do 🧐

Manarnz replied:
I don't know what it does. I think maybe I should stick to the task? What do you think?

YesugenB replied:
Yeah it's fine! No need to change then

YesugenB reviewed (approved):
Looks good, just had a small question/suggestion that's not very related to this. 

Btw in AssignmentsTable.js line 43 Epoch is misspelled as Epcoch.

Manarnz committed:
Revert "Renamed absTime in LinkBB to absTP"

This reverts commit cb74b15a401c0aabe30eb700a4d8eabd3b181365.

Manarnz committed:
Assigned absTime with myNull

amgrubb commented:
@Manarnz  Do AssignmentsTable.js lines 400 and 425 need to be updated with myNull as well?

Manarnz committed:
Made changes to lines 400 and 421 in AssignmentsTable.js

Manarnz commented:
> @Manarnz Do AssignmentsTable.js lines 400 and 425 need to be updated with myNull as well?

I have checked these two lines and yes they do need change. For line 400, I removed the if statement as the TODO comment suggested.

amgrubb commented on leaf-ui/rappid-extensions/AssignmentsTable.js:
```diff
@@ -395,11 +395,7 @@ var LinkRelationshipView = Backbone.View.extend({
-        if (this.model.get('absTime') == -1) {
-            this.$('.linkAbsRelation').val('')
-        }
+        this.$('.linkAbsRelation').val('')
```
The `if` statement must remain because otherwise save values will not appear in the view. Switch the original if statement to `myNull` instead of `-1`.

amgrubb reviewed (changes requested):
We need to put the `if` statement back.

Manarnz committed:
Implemented Alicia's comment

amgrubb commented:
Please address or resolve the conversations above, I'll review it after that. 😄

angelac2023 committed:
Fixed Absolute Time Point Bug

When the user creates links between intentions, the assignment table no longer displays their absolute time points as the same value.

angelac2023 committed:
Fixed Typo

Changed "Epcoch" to "Epoch" in Line 44 of AssignmentsTable.js

angelac2023 committed:
Delete app.js

angelac2023 committed:
Revert "Delete app.js"

This reverts commit 0007f8cc2d081acb68da1b3e38e3830604991524.

angelac2023 committed:
Implementing Manar's comments

Removed unnecessary comment and restored app.js

amgrubb commented:
- [x] There is still an issue, when you click `Unassign` for one relationship, it unassigns all of them.

angelac2023 committed:
Fixed Bug in Unassign Abs Time

Fixed bug where if user tries to unassign one absolute time point, they unassign for all links.

amgrubb reviewed (approved):
Excellent work!

amgrubb committed:
Merge branch 'backbone-refactor' into maac-issue493
